pkg/redshift/routes/routes.go

```diff
@@ -33,21 +33,21 @@ func (r *RedshiftResourceHandler) secret(rw http.ResponseWriter, req *http.Reque
 	routes.SendResources(rw, secret, err)
 }
 
-func (r *RedshiftResourceHandler) cluster(rw http.ResponseWriter, req *http.Request) {
+func (r *RedshiftResourceHandler) clusters(rw http.ResponseWriter, req *http.Request) {
 	reqBody, err := routes.ParseBody(req.Body)
 	if err != nil {
 		rw.WriteHeader(http.StatusBadRequest)
 		routes.Write(rw, []byte(err.Error()))
 		return
 	}
```

Comment:
you don't need now to parse the body, it should be a GET request (similar to the `Secrets` function)